default.nix

```diff
@@ -1,66 +1,121 @@
-{ frontend ? false }:
+{ obelisk ? import ./.obelisk/impl {}
+, pkgs ? obelisk.nixpkgs, ... } @ args:
+
 let
-  obelisk-src = (import <nixpkgs> {}).fetchFromGitHub {
-    owner = "obsidiansystems";
-    repo = "obelisk";
-    rev = "6c38599615eddba1b9f8dfb845f7404f53ed8053";
-    sha256 = "0pqhppn2cb69v7r6wbg2zrx5ylxbgq7bl7qilarfbmyd9gcph9h4";
-  };
-  reflex-platform = (import obelisk-src {}).reflex-platform;
 
-  gargoyle-src = reflex-platform.nixpkgs.fetchFromGitHub {
-    owner = "obsidiansystems";
-    repo = "gargoyle";
-    rev = "2c19c569325ad76694526e9b688ccdbf148df980";
-    sha256 = "0257p0qd8xx900ngghkjbmjnvn7pjv05g0jm5kkrm4p6alrlhfyl";
-  };
-  groundhog-src = reflex-platform.nixpkgs.fetchFromGitHub {
-    owner = "obsidiansystems";
-    repo = "groundhog";
-    rev = "c2f18be45e3233f6268c8468eb0732dd6b2e8009";
-    sha256 = "1r9i78bsnm6idbvp87gjklnr10g7c83nsbnrffkyrn1wmd7zzqdn";
+  reflex-platform = obelisk.reflex-platform;
+  inherit (pkgs) lib;
+  haskellLib = pkgs.haskell.lib;
+
+  # Some dependency thunks needed
+  repos = {
+
+    # Not sure why this is needed?
+    groundhog = pkgs.fetchFromGitHub {
+      owner = "obsidiansystems";
+      repo = "groundhog";
+      rev = "febd6c12a676693b1d7339e54a4d107c4a67fcc3";
+      sha256 = "1q05nrqdzh26r17wsd53sdj106dxh3qlg66pqr3jsi8d63iyaq8k";
+    };
+
+    # bytestring-trie in hackage doesn’t support base 4.11+
+    bytestring-trie = pkgs.fetchFromGitHub {
+      owner = "obsidiansystems";
+      repo = "bytestring-trie";
+      rev = "27117ef4f9f01f70904f6e8007d33785c4fe300b";
+      sha256 = "103fqr710pddys3bqz4d17skgqmwiwrjksn2lbnc3w7s01kal98a";
+    };
+
+    # Rhyolite needs throttleBatchWithLag from Reflex.Time
+    reflex = pkgs.fetchFromGitHub {
+      owner = "reflex-frp";
+      repo = "reflex";
+      rev = "2e2e19d37a5f1151fe985190b479016640751578";
+      sha256 = "0ygp3firh2h770wsfpzykph8jlb3xpi3bdl0imqa6zgcbphf13vx";
+    };
+
+    # Reflex needs monoidal-containers 0.4.0.0 but that is not
```

Comment:
If you bump to a recent `reflex-platform` this should be included though that may require sending a bump to obelisk first

Reply:
Yeah, I think we might also want projects to use it with older obelisks?